project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=0.13.16
+sbt.version=1.0.0
```

Comment:
I'm eager to update the sbt build. It would be awesome if you could do that in a separate pull request.